docs/getting-started/configuration.md

```diff
@@ -187,6 +187,56 @@ components: {
  },
 ```
 
+Since [v1.1.0](https://github.com/backstage/backstage/releases/tag/v1.1.0-next.3), you must provide an explicit sign-in resolver.
+
+Open `packages/backend/src/plugins/auth.ts` and replace with following code snippet:
```

Comment:
thanks for the review, I think it may depend on personal learning habit, I personally prefer to have something running first, then tweak some pieces to see the change -> in order to understand more about the system. Also because I don't have much knowledge about nodejs or  "what's identity resolver". In this case, I feel I need this example to have a runnable backstage in local.

What do you think? Any chance the sample in this PR might be outdated? If so, I agree we may not want it

Reply:
Yeah it's a bit tough to keep it all up to date. But as a general rule, we hope to get the flow of reading in this doc in such a way that it's not too much code interspersed with other important information so that it reads better as a whole. If it's a piece of code without the context to understand even why you do it, that can pull down the value of it a bit too. Yeah I think we appreciate the intent here but that it might be better to link to an article than having the code inline.